src/Sylius/Behat/Context/Api/Shop/ProductContext.php

```diff
@@ -83,6 +83,30 @@ public function iSearchForProductsWithName(string $name)
         $this->client->filter();
     }
 
+    /**
+     * @Then I should see :amount product reviews
+     */
+    public function iShouldSeeProductReviews(int $amount): void
+    {
+        Assert::count($this->responseChecker->getValue($this->client->getLastResponse(), 'reviews'), $amount);
```

Comment:
in responseChecker we have function `countCollection`, you can use also this function instead `count`( i guess :D)

Reply:
yeah, it would work for normal collection, in this case we have collection of subresources